tests/unit/cmd/sca/test_scan.py

```diff
@@ -115,8 +115,7 @@ def test_sca_scan_diff(client: GGClient, dummy_sca_repo: Repository):
         result = sca_scan_diff(
```

Comment:
Not related to this line: could you add a unit test (ie: with a cassette) for the pre-receive scan ? IMO it will be useful for debugging in case we have an issue with the functional one (it will show a change in the API)

Reply:
I agree. I think one with a vuln and one without would be great.